how-it-works/5_governance/sns.card.md

```diff
@@ -8,6 +8,7 @@ coverImage: /img/how-it-works/network-nervous-system-nns.600x300.jpg
 
 # Service Nervous System (SNS)
 
-An SNS would derive from the [Network Nervous System (NNS)](/how-it-works/network-nervous-system-nns/), the autonomous tokenized governance system that controls the Internet Computer blockchain in a completely open, permissionless, and decentralized manner.
+Similarly to how the Internet Computer (IC) is controlled by the Network Nervous System, an individual application on the IC (called *dapp*) can be controlled by a decentralized community. The Internet Computer’s built-in solution for a DAO (decentralized autonomous organization) that *tokenizes* and *decentralizes* a dapp is called the Service Nervous System (SNS). In the process of creating an SNS, new tokens are minted and sold in a community-based fundraising. The dapp’s control is handed over to the SNS and everyone who has SNS tokens can contribute to decisions on how the dapp evolves going forward. 
+This allows distributing the power and ownership of the dapp over a number of parties, thereby eliminating single points of failure and gaining censorship resistance. Moreover, the newly created tokens can be used to create incentives to foster user adoption and participation.
 
-[Go deeper](/how-it-works/sns/)
\ No newline at end of file
+[Go deeper](/how-it-works/sns/](https://internetcomputer.org/docs/current/tokenomics/sns/sns-intro-tokens/)
```

Comment:
If I go to the preview, this link doesn't work for me. Do you have the same @bjoernek ?

Reply:
Yes, I have the same (and the same also applies to previews for existing cards). So I assume that we should be fine.

Reply:
@sesi200 : Could you kindly confirm that using including the following like also looks fine for you ?
[Go deeper](/docs/current/tokenomics/sns/sns-intro-tokens/)

Reply:
yep, looks good